src/librustdoc/lib.rs

```diff
@@ -391,7 +396,14 @@ fn main_args(args: &[String]) -> isize {
     let diag_opts = (options.error_format,
                      options.debugging_options.treat_err_as_bug,
                      options.debugging_options.ui_testing);
+    let show_coverage = options.show_coverage;
     rust_input(options, move |out| {
```

Comment:
Is it necessary to call `rust_input` in `show_coverage` case?

Reply:
`rust_input` is what actually calls the `core::run` documentation loading code, so i wanted to go through that to make sure that it used the same cleaning/filtering process that regular docs loading did.

Reply:
Seems a bit overkill but let's go along for now...